entrypoint.sh

```diff
@@ -48,9 +59,15 @@ if [ -z "$TV_NAME" ]; then
 fi
 echo "Found TV name: $TV_NAME"
 
-echo "Attempting to install jellyfin-tizen-builds $JELLYFIN_BUILD_OPTION.wgt from release: $TAG"
-echo "$DOWNLOAD_URL"
+echo "Downloading jellyfin-tizen-builds $JELLYFIN_BUILD_OPTION.wgt from release: $TAG"
+wget -q --show-progress "$DOWNLOAD_URL"; echo ""
 
-wget -q --show-progress "$DOWNLOAD_URL"
+if ! [ -z "$CERTIFICATE_PASSWORD" ]; then
+	echo "Attempting to sign package using provided certificate"
+	sed -i "s/_CERTIFICATEPASSWORD_/$CERTIFICATE_PASSWORD/" profile.xml
+	sed -i '/<\/profile>/ r profile.xml' /home/developer/tizen-studio-data/profile/profiles.xml
+	tizen package -t wgt -s custom -- $JELLYFIN_BUILD_OPTION.wgt
```

Comment:
It'd be nice to verify the signing attempt worked. It doesn't look like tizen returns valid exit codes, but you can check the the last file modification times to see if's worked:
```
       package_mtime=$(stat -c %Y $JELLYFIN_BUILD_OPTION.wgt)
        tizen package -t wgt -s custom -- $JELLYFIN_BUILD_OPTION.wgt
       signed_package_mtime=$(stat -c %Y $JELLYFIN_BUILD_OPTION.wgt)
       [ $package_mtime -eq $signed_package_mtime ] && echo "Error signing package. Verify your certificate password is correct." && exit 1
```

Reply:
I like the idea, I'm unable to test it now so I think we can get this merged as is and you can send a PR with the change.